app.js

```diff
@@ -17,13 +17,18 @@ import paymentRouter from "./routes/payment";
 import conversationRouter from "./routes/conversation";
 
 var app = express();
+const path = require("path")
```

Comment:
@ykurata In our previous code we had 
`var socket_io = require( "socket.io" );
var io = socket_io();
app.io = io;
const path = require("path")`

Do we not need the rest of this now?

Reply:
I think we don't need that part in app.js